### PR TITLE
Minor shell script tweaks

### PR DESCRIPTION
**Pull request by XhmikosR**
* always double quote variables
* use https for the miniconda URLs

Would be nice to try this without the cache just in case.

### CODE REVIEW

File: ci/install_conda.sh [hunk 1]
```diff
@@ -1,16 +1,16 @@
 #!/bin/bash
 
 if [ -d "$HOME/miniconda3" ] && [ -e "$HOME/miniconda3/bin/conda" ]; then
-    echo "Miniconda install already present from cache: $HOME/miniconda3"
-    rm -rf $HOME/miniconda3/envs/hosts  # Just in case...
+    echo "Miniconda install already present in cache: $HOME/miniconda3"
+    rm -rf "$HOME"/miniconda3/envs/hosts  # Just in case...
```
Comment (funilrys) [inline ci/install_conda.sh]: The double quote is not necessary for our use case.

File: ci/setup_conda_env.sh [hunk 2]
```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 echo "Creating a Python $PYTHON_VERSION environment"
-conda create -n hosts python=$PYTHON_VERSION || exit 1
+conda create -n hosts python="$PYTHON_VERSION" || exit 1
```
Comment (funilrys) [inline ci/setup_conda_env.sh]: Not necessary too.

**Comment by funilrys** [comment]
> Would be nice to try this without the cache just in case. 

Well, it would be great but it's not necessary if we just switch to https.

File: ci/install_conda.sh (hunk 1 above)
Comment (XhmikosR) [reply ci/install_conda.sh]: Even if it's not necessary I went with consistency in this patch.

**Comment by XhmikosR** [comment]
> Your fork normally doesn't have any cache files. So you can test from your fork. Just activate the repository in your Travis CI settings.

Sorry, not going to install Travis CI just for this. Ideally we should just switch to GitHub Actions, but that's for another time.

**Comment by funilrys** [comment]
> Sorry, not going to install Travis CI just for this. Ideally we should just switch to GitHub Actions, but that's for another time.

That won't change today! It's just a switch for your fork inside your Travis CI settings... 

Anyway, It's yours Steve @StevenBlack.

**Comment by XhmikosR** [comment]
It's probably a matter of a few minutes for me to submit a PR in the next days switching to GitHub Actions CI, as long as it works and will be accepted by @StevenBlack  :)

**Comment by funilrys** [comment]
The problem is that later we will have to implement an auto push/release in order to reduce the work of Steve @StevenBlack. It might be easier to do from Travis CI. 

In the current state, it's just for testing the source code so it works well!

**Comment by XhmikosR** [comment]
You can do anything with GitHub Actions CI :) Plus your secrets stay in the repo settings which is a huge plus for me.

**Comment by funilrys** [comment]
My secrets in Travis CI stay encrypted in the configuration file ... But we are talking about cosmetics. 

The most important is that others understand it, and it works without having to change a lot for the maintainer itself.

**Comment by StevenBlack** [comment]
I'm good with this one, too.  Thank you @XhmikosR and Nissar @funilrys too.

Merging.

**Pull request merged**